lib/galaxy/tools/parameters/basic.py

```diff
@@ -151,7 +151,7 @@ def value_to_display_text( self, value, app ):
         Convert a value to a text representation suitable for displaying to
         the user
         """
-        if value:
+        if value is not None:
             return unicodify( value )
         return "Not available."
```

Comment:
Booleans always default to 'false' or 'true', but I agree that we should fix this for integers and empty strings and I a appreciate that you are looking to it. It would be nice to have a solution where always something is displayed. "''" would probably do to for empty strings.